Shared/Samples/Show viewshed from geoelement in scene/ShowViewshedFromGeoelementInSceneView.swift

```diff
@@ -0,0 +1,200 @@
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+import ArcGIS
+
+struct ShowViewshedFromGeoelementInSceneView: View {
+    /// The view model for the sample.
+    @StateObject private var model = Model()
+    
+    /// The timer for the moving tank animation.
+    @State private var animationTimer: Timer!
+    
+    var body: some View {
+        SceneView(
+            scene: model.scene,
+            cameraController: model.cameraController,
+            graphicsOverlays: [model.graphicsOverlay],
+            analysisOverlays: [model.analysisOverlay]
+        )
+        .onSingleTapGesture { _, mapPoint in
+            // Start a timer to animate the tank moving towards the new waypoint.
+            model.waypoint = mapPoint
+            animationTimer = Timer.scheduledTimer(withTimeInterval: 0.1, repeats: true) { _ in
+                model.animate()
+                if model.waypoint == nil {
+                    animationTimer.invalidate()
+                }
+            }
+        }
+        .overlay(alignment: .top) {
+            // Instruction text.
+            Text("Tap on the map to move the tank and update the viewshed.")
+                .padding(8)
+                .frame(maxWidth: .infinity, alignment: .leading)
+                .background(.thinMaterial, ignoresSafeAreaEdges: .horizontal)
+        }
+    }
+}
+
+private extension ShowViewshedFromGeoelementInSceneView {
+    /// The view model for the sample.
+    class Model: ObservableObject {
+        /// A scene with an imagery basemap.
+        let scene: ArcGIS.Scene = {
+            let scene = Scene(basemapStyle: .arcGISImagery)
+            
+            // Add elevation source to the base surface of the scene with the service URL.
+            let elevationSource = ArcGISTiledElevationSource(url: .worldElevationService)
+            scene.baseSurface.addElevationSource(elevationSource)
+            
+            // Create the building layer and add it to the scene.
+            let buildingsLayer = ArcGISSceneLayer(url: .brestBuildingsService)
+            scene.addOperationalLayer(buildingsLayer)
+            
+            // Set scene the viewpoint specified by the camera position.
+            let point = Point(x: -73.0870, y: -49.3460, z: 5046, spatialReference: .wgs84)
+            let camera = Camera(location: point, heading: 11, pitch: 62, roll: 0)
+            scene.initialViewpoint = Viewpoint(boundingGeometry: point, camera: camera)
```

Comment:
I don't think these lines have any effect on the sample - the starting viewpoint is determined by the camera controller.